spaceship/lib/spaceship/client.rb

```diff
@@ -3,7 +3,6 @@
 require 'faraday_middleware'
 require 'fastlane/version'
 require 'logger'
-require 'spaceship/babosa_fix'
```

Comment:
Do we need to replace this with a `require 'babosa'` if we're taking out the fix class?